Gopkg.toml

```diff
@@ -19,36 +19,70 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.6"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.6"
 
 [[constraint]]
   name = "k8s.io/code-generator"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.6"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.11.3"
+  version = "kubernetes-1.12.6"
+
+[[constraint]]
+  name = "github.com/knative/caching"
+  # HEAD as of 2019-03-14
+  revision = "b9d80eaec52c6ee5841271438cb020a80be72cb2"
 
 [[override]]
-  name = "github.com/json-iterator/go"
-  # This is the commit at which k8s depends on this in 1.11
-  # It seems to be broken at HEAD.
-  revision = "f2b4162afba35581b6d4a50d3b8f34e33c144682"
+  name = "github.com/knative/pkg"
+  # HEAD as of 2019-03-19
+  revision = "4840bbe7b47ff2f6abb4bbf7d023bf98c0a51b6a"
 
 [[override]]
-  name = "github.com/Azure/go-autorest"
-  # This is the commit at which k8s depends on this in 1.11
-  # It seems to be broken at HEAD.
-  revision = "1ff28809256a84bb6966640ff3d0371af82ccba4"
+  name = "contrib.go.opencensus.io/exporter/stackdriver"
+  # HEAD as of 2019-02-11
+  # Needed because this includes a fix to support Stackdriver built-in metrics
+  revision = "c06c82c832edca4eaf7b0241bd655560a1be0346"
 
 [[override]]
-  name = "github.com/knative/pkg"
-  # HEAD as of 2019-02-14 💖
-  revision = "0183bf9cdc7349e3c76c26e51f84689b411fccea"
+  name = "github.com/golang/protobuf"
+  # What serving has in Gopkg.lock
+  revision = "aa810b61a9c79d51363740d207bb46cf8e620ed5"
+
+[[override]]
+  name = "google.golang.org/grpc"
+  # What serving has in Gopkg.lock
+  revision = "2e463a05d100327ca47ac218281906921038fd95"
+
+[[override]]
+  name = "google.golang.org/api"
+  # What serving has in Gopkg.lock
+  revision = "511bab8e55decfbe784cd2d1868fa5cda62dc6c1"
+
+[[override]]
+  name = "github.com/census-instrumentation/opencensus-proto"
+  # What serving has in Gopkg.lock
+  revision = "7f2434bc10da710debe5c4315ed6d4df454b4024"
+
+[[override]]
+  name = "golang.org/x/oauth2"
+  # What serving has in Gopkg.lock
```

Comment:
This comment will get out of date almost immediately. Is there a better way to describe why these overrides are here? Why do we need them?

Reply:
Now that these are locked, we can actually just remove these...  🙄